modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -1340,19 +1445,34 @@ GrainTracker::boundingRegionDistance(std::vector<MeshTools::BoundingBox> & bboxe
   return min_distance;
 }
 
+unsigned int
+GrainTracker::getNextUniqueID()
+{
+  /**
+   * Get the next unique grain ID but make sure to respect
+   * reserve ids.
+   */
+  while (++_max_curr_grain_id <= _reserve_grain_first_idx + _n_reserve_ops)
```

Comment:
Nope - the +1 isn't necessary in the second expression because `_reserve_grain_first_idx` is a valid index.

e.g.
Let's say I have 42 grains and I have 2 reserved ops. My grains will have indices from [0, 41]. The reserve grain indices will be 42 and 43. The first time I assign a move a new grain from the reserve op, it needs to be index number 44. I assign `_reserve_grain_first_idx` in line 485:

```
    _reserve_grain_first_idx = _max_curr_grain_id + 1;
   // 42 = 41 + 1
```
Therefore the first index for a new grain is _reserve_grain_first_idx (42) + _n_reserve_ops (2).